server/src/main/java/com/objectcomputing/checkins/services/skills/SkillController.java

```diff
@@ -1,135 +1,137 @@
 package com.objectcomputing.checkins.services.skills;
 
+import com.objectcomputing.checkins.services.role.RoleType;
+import io.micronaut.http.HttpRequest;
 import io.micronaut.http.HttpResponse;
 import io.micronaut.http.HttpStatus;
 import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Error;
 import io.micronaut.http.annotation.*;
+import io.micronaut.http.hateoas.JsonError;
+import io.micronaut.http.hateoas.Link;
 import io.micronaut.security.annotation.Secured;
 import io.micronaut.security.rules.SecurityRule;
 import io.swagger.v3.oas.annotations.tags.Tag;
 
 import javax.annotation.Nullable;
 import javax.inject.Inject;
 import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
 import java.net.URI;
 import java.util.List;
 import java.util.Set;
 import java.util.UUID;
 
 @Controller("/services/skill")
-@Secured(SecurityRule.IS_ANONYMOUS)
+@Secured(SecurityRule.IS_AUTHENTICATED)
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
-@Tag(name="skill")
+@Tag(name = "skill")
 public class SkillController {
 
     @Inject
-    private SkillServices skillServices;
+    private SkillServicesImpl skillServices;
```

Comment:
is there a reason that we are injecting the implementation instead on interface ?

Reply:
Agreed, change it back please.